CHANGELOG.md

```diff
@@ -22,6 +22,8 @@
  - Adding standard for `SpriteComponent` and `SpriteAnimationComponent` constructors
  - Move files to comply with the dart package layout convention
  - Fix gesture detection bug of children of `PositionComponent`
+ - The `game` argument on `GameWidget` is now required
+ - Added a note to the `GameWidget` docs mentioning to not instantiate inside a build method
```

Comment:
I don't think we need to be this finegrained in the changelog?

Reply:
Indeed, I went a bit overboard with adding entries haha.